packages/react-components/react-spinbutton/src/components/SpinButton/SpinButton.types.ts

```diff
@@ -29,14 +29,17 @@ export type SpinButtonSlots = {
   decrementButton: NonNullable<Slot<'button'>>;
 };
 
-export type SpinButtonCommons = {
+/**
+ * SpinButton Props
+ */
+export type SpinButtonProps = Omit<ComponentProps<Partial<SpinButtonSlots>, 'input'>, 'onChange' | 'size'> & {
   /**
    * Initial value of the control (assumed to be valid). Updates to this prop will not be respected.
    *
    * Use this if you intend for the SpinButton to be an uncontrolled component which maintains its
    * own value. For a controlled component, use `value` instead. (Mutually exclusive with `value`.)
    */
-  defaultValue: number;
+  defaultValue?: number;
```

Comment:
I've been putting the props in alphabetical order, if you don't mind doing that here too :)